geo-types/CHANGES.md

```diff
@@ -2,8 +2,10 @@
 
 ## Unreleased
 
+* BREAKING: Make `Rect::to_lines` return lines in winding order for `Rect::to_polygon`.
```

Comment:
This was probably oversight. Good catch! 

I personally don't think this is the kind of change we'd bump semantic versioning for. Does anyone disagree? 

I'm ok to leave "BREAKING" or whatever in the release notes, I'm just loathe to do anything that causes all our downstream crate users to be on an old version.

Reply:
I don't have any context on downstream consumers yet, so I don't have a clear opinion here.

I think this falls in the "Possibly-breaking change" per [cargo book](https://doc.rust-lang.org/cargo/reference/semver.html) and in unstable packages, it's OK to bump just the patch version for these changes. The order of returned lines isn't part of an explicit contract, so someone depending on that order to *not* conform to the winding order is unlikely.

Let me know if I can help make this decision (perhaps start a thread on the channel?)

Reply:
In the interest of a certain amount of efficiency, I don't expect to get an explicit "OK" on all things. Typically when I'm about to do something potentially controversial, my approach is to make a proposal in the code review and solicit input. If no one complains (we're all async, so I wait a couple days) I go forward. 

If it were something that seemed potentially *very* controversial, I might wait for longer or for a more explicit OK from a larger group of people.

Since no one has yet replied to the contrary, I estimate that it's reasonable to move forward without bumping major version at this point.